README.md

```diff
@@ -1,29 +1,570 @@
+    <http://example.org/ld-experts> <http://schema.org/name> "LD Experts" .
+    _:b0 <http://schema.org/image> <http://manu.sporny.org/images/manu.png> .
+    _:b0 <http://schema.org/name> "Manu Sporny" .
+    _:b0 <http://schema.org/url> <http://manu.sporny.org/> .
+    _:b0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
+
+    */
+}
+```
+
+### <a name="fromrdf"></a>fromRDF (N-Quads)
+
+```csharp
+var opts = new JsonLdOptions();
+var jsonld = JsonLdProcessor.FromRDF(serialized, opts);
+Console.WriteLine(jsonld);
+
+/*
+
+Output:
+[
+    {
+        "@id": "_:b0",
+        "http://schema.org/image": [
+            {
+                "@id": "http://manu.sporny.org/images/manu.png"
+            }
+        ],
+        "http://schema.org/name": [
+            {
+                "@value": "Manu Sporny"
+            }
+        ],
+        "http://schema.org/url": [
+            {
+                "@id": "http://manu.sporny.org/"
+            }
+        ],
+        "@type": [
+            "http://schema.org/Person"
+        ]
+    },
+    {
+        "@id": "http://example.org/ld-experts",
+        "http://schema.org/member": [
+            {
+                "@id": "_:b0"
+            }
+        ],
+        "http://schema.org/name": [
+            {
+                "@value": "LD Experts"
+            }
+        ]
+    }
+]
+*/
+```
+
+_or_ using a custom RDF parser
+
+```csharp
+private class CustomRDFParser : IRDFParser
+{
+    public RDFDataset Parse(JToken input)
+    {
+        // by public decree, references to example.org are normalized to https going forward...
+        var converted = ((string)input).Replace("http://example.org/", "https://example.org/");
+        return RDFDatasetUtils.ParseNQuads(converted);
+    }
+}
+
+internal static void Run()
+{
+    var parser = new CustomRDFParser();
+    var jsonld = JsonLdProcessor.FromRDF(_serialized, parser);
+    Console.WriteLine(jsonld);
+
+    /*
+
+    Output:
+    [
+        {
+            "@id": "_:b0",
+            "http://schema.org/image": [
+                {
+                    "@id": "http://manu.sporny.org/images/manu.png"
+                }
+            ],
+            "http://schema.org/name": [
+                {
+                    "@value": "Manu Sporny"
+                }
+            ],
+            "http://schema.org/url": [
+                {
+                    "@id": "http://manu.sporny.org/"
+                }
+            ],
+            "@type": [
+                "http://schema.org/Person"
+            ]
+        },
+        {
+            "@id": "https://example.org/ld-experts",
+            "http://schema.org/member": [
+                {
+                    "@id": "_:b0"
+                }
+            ],
+            "http://schema.org/name": [
+                {
+                    "@value": "LD Experts"
+                }
+            ]
+        }
+    ]
+    */
+}
+```
+
+
+### Custom documentLoader
```

Comment:
```suggestion
### Custom DocumentLoader
```

Reply:
the casing of the property is unfortunately starting with a lowercase letter. These titles in the json-ld.js repository are intended to align with the exact object member you would use to invoke the behaviour. However I guess we've moved away from that above, so no harm in altering this one too.